tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { useState, useEffect, useRef } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
-import { fetchStts } from '../../actions/stts'
+import { Redirect } from 'react-router-dom'
+
+import { fetchSttList } from '../../actions/sttList'
+import { requestAccess } from '../../actions/requestAccess'
+
 import Button from '../Button'
+import FormGroup from '../FormGroup'
```

Comment:
I pulled out the `inputs` in this page because they were very similar. This is the import for that new `FormGroup` component.